src/__mocks__/api-handlers.js

```diff
@@ -13,202 +13,240 @@ const httpCreated = 201
 const httpAuthorizationRequired = 401
 const httpNotFound = 404
 
-export const handlers = [
-  rest.get('https://api.github.com/repos/:org/:repo/issues', (req, res, ctx) => {
-    const {org, repo} = req.params
-
-    if (org !== 'pablo-mayrgundter' || repo !== 'Share') {
-      return res(ctx.status(httpNotFound))
-    }
-
-    return res(
-        ctx.status(httpOk),
-        ctx.json(MOCK_ISSUES.data),
-    )
-  }),
-
-  rest.get('https://api.github.com/repos/:org/:repo/issues/:issueNumber/comments', (req, res, ctx) => {
-    const {org, repo, issueNumber} = req.params
-
-    if (org !== 'pablo-mayrgundter' || repo !== 'Share' || !issueNumber) {
-      return res(ctx.status(httpNotFound))
-    }
-
-    return res(
-        ctx.status(httpOk),
-        ctx.json(MOCK_COMMENTS.data),
-    )
-  }),
-
-  rest.get('https://api.github.com/repos/:org/:repo/contents/:path', (req, res, ctx) => {
-    const {org, repo, path} = req.params
-    const ref = req.url.searchParams.get('ref')
-
-    if (org !== 'bldrs-ai' || repo !== 'Share' || path !== 'README.md') {
+
+/**
+ * Initialize API handlers, including Google Analytics and GitHub.
+ *
+ * @return {Array<object>} handlers
+ */
+export function initHandlers() {
```

Comment:
Why do these need to be functions?

Is there a reason the spread operator can't be used here for concatenating two arrays?

Reply:
They were defined at root scope, so to use spreading (my first impulse too) would mean joining them at the end of the file, which I thought was less readable.